plugins/modules/lvg.py

```diff
@@ -632,7 +743,7 @@ def main():
         argument_spec=dict(
             state=dict(type='str', choices=['absent', 'present', 'varyoff', 'varyon'], default='present'),
             vg_name=dict(type='str', required=True),
-            vg_type=dict(type='str', choices=['none', 'big', 'scalable']),
+            vg_type=dict(type='str', choices=['small', 'big', 'scalable', None]),
```

Comment:
so "small" is added to the default VG type in vg_type dict, but I don't see "small" is handled at line 374 vg_type_opt. are you doing this on purpose?

Reply:
If we decide to add small to vg type option, we need to update the documentation at line 50

Reply:
i will be removing 'small' in here. it is some leftover, that i missed to cleanup during the review we did on the video. thanks for picking it up